### ISSUE TITLE
Issue_3

### ISSUE THREAD

**Issue by Tishk01** [Sarah-Eschmann/MALIS-21.3--WPM_T9_Sarah_Eschmann#3]
Werden die Daten tatsächlich in jedem Monat benötigt, also schaut sich das jemand jeden Monat an? Ansonsten könnte man den Rythmus ja auch verändern und z. B. einmal/zweimal/viermal jährlich mehrere Monate zusammen abrufen und auswerten. und damit den Arbeitsaufwand reduzieren.

**Comment by Sarah-Eschmann** [comment]
Ja, leider will die Direktion die Daten tatsächlich pünktlich jeden Monat haben.
Die Idee, die Daten quartalsweise auszuwerten finde ich aber sehr gut! Vielleicht kann ich unsere Direktion ja davon überzeugen, da das den Arbeitsaufwand und somit auch Kosten verringern würde.
